lib/routes/spglobal/ratings.ts

```diff
@@ -0,0 +1,78 @@
+import { Route, ViewType } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import got from '@/utils/got';
+
+export const route: Route = {
+    path: '/ratings/:language?',
+    categories: ['finance', 'popular'],
+    view: ViewType.Articles,
+    example: '/ratings/en/',
+    parameters: {
+        language: {
+            description: '语言',
+            options: [
+                { value: 'zh', label: '中文' },
+                { value: 'en', label: 'English' },
+                { value: 'es', label: 'Español' },
+                { value: 'pt', label: 'Português' },
+                { value: 'jp', label: '日本語' },
+                { value: 'ru', label: 'Русский' },
+                { value: 'ar', label: 'العربية' },
+            ],
+        },
+    },
+    features: {
+        requireConfig: [
+            {
+                name: 'LANGUAGE',
+                description: '语言',
+            },
+        ],
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.spglobal.com/ratings/:language'],
+        },
+    ],
+    name: 'S&P Global Ratings',
```

Comment:
Do not include namespace name https://github.com/DIYgod/RSSHub/blob/a7ac1c5d4b026f6fd98d199f3cd440475e4c3b1e/lib/routes/spglobal/namespace.ts#L4 in route name.